src/extensions/Statiq.Razor/RenderRazor.cs

```diff
@@ -41,6 +41,7 @@ public class RenderRazor : Module
         private Config<NormalizedPath> _viewStartPath;
         private Config<NormalizedPath> _layoutPath;
         private Config<object> _model;
+        private IDictionary<string, Config<object>> _viewData = null;
```

Comment:
👏 storing the items as `Config<object>` and then resolving them at execution is great, good call